packages/terra-table/NOTICE

```diff
@@ -0,0 +1,13 @@
+Copyright 2017 - 2019 Cerner Innovation, Inc.
```

Comment:
Is this correct now that we are Oracle?

Reply:
We should update the year at least.

Reply:
Updated the copyright for year and company.